### ISSUE TITLE
tutum/curl has disappeared

### ISSUE THREAD

**Issue by dvejmz** [docker/hub-feedback#1672]
## Problem description

The Tutum cURL Docker Hub page has been removed. The [page](https://hub.docker.com/r/tutum/curl/) 404's and the image is no longer available. I'm unsure if this could be related to the [recent Docker Registry issue](https://status.docker.com/pages/533c6539221ae15e3f000031) Docker Hub experienced that appears to be resolved now or is something different.

Bonus: http://www.tutum.co/ is down, getting 403 errors :)

**Comment by nandhini915** [comment]
Hi @dvejmz -- removing the tutum/curl repository was intentional, it is part of the deprecation of hosted cluster management feature provided on cloud.docker.com. tutum.co was also taken down for the same reasons. 
https://success.docker.com/article/cloud-migration is an article referencing this change, which was announced on March 21, we'd also emailed all users of the service informing them about the change. 

Please let us know if you have more questions.

**Comment by dvejmz** [comment]
I understand that. However, on that announcement you linked I see no mention of the taking down of some or all of the Docker repos hosted on https://hub.docker.com/r/tutum/. This may affect many more users than just those using Docker Cloud, as the repositories hosted under this account don't really have much to do with Docker Cloud itself.

Anyway, we'll be moving away from all images provided by tutum now.

**Comment by nandhini915** [comment]
Thanks for the additional context @dvejmz -- we did intend to take out all content connected with the tutum/ namespace to make the transition completely clear. It was a miss on our side that we did not communicate this well enough. 
The curl image was made private, I've updated it to be a public image again so your workflows can continue unaffected - we do recommend moving away from this image since we expect to take this down soon.
